.github/workflows/shippable_builds.yml

```
@@ -409,9 +446,20 @@ jobs:
       - name: Setup Gradle
         uses: gradle/actions/setup-gradle@v4
         with:
-          cache-disabled: ${{ needs.dump_config.outputs.releaseType == 'beta' || needs.dump_config.outputs.releaseType == 'release' }}
+          cache-disabled: "${{ contains(fromJSON('[\"beta\", \"release\"]'), needs.dump_config.outputs.releaseType) }}"
           add-job-summary: on-failure
 
+      - name: Set Version Code for Daily
+        shell: bash
+        env:
```

Shouldn't this be daily only?
```suggestion
      - name: Set Version Code for Daily
        if: ${{ needs.dump_config.outputs.releaseType == 'daily' }}
        shell: bash
        env:
```